omnipaxos/src/ballot_leader_election.rs

```diff
@@ -99,16 +100,11 @@ pub(crate) struct BallotLeaderElection {
     /// Vector which holds all the received heartbeats
     ballots: Vec<BLEBallot>,
     /// Holds the current ballot of this instance.
-    current_ballot: Ballot, // (round, pid)
-    /// The number of replicas inside the cluster that this instance is connected to (based on
-    /// heartbeats received) including itself.
-    connectivity: u8,
+    current_ballot: BLEBallot,
```

Comment:
As a result of the previous comment regarding making `BLEBallot` private, it probably is better to bring back `current_ballot: Ballot` and `connectivity: u8 ` as fields in BLE. I also believe that it will make the code for BLE more understandable, since we can easier see what fields are in BLE rather than them being hidden behind `BLEBallot`.